docs/modules/ROOT/pages/how-to-guides/proc_prefetching-dependencies-to-support-hermetic-build.adoc

```diff
@@ -201,6 +203,39 @@ spec:
 
 include::../partials/con_hermetic_verification.adoc[]
 
+== Enabling prefetch builds for `npm`
+Cachi2 supports `npm` by fetching any dependencies you declare in your `package.json` and `package-lock.json` project files. The npm CLI manages the `package-lock.json` file automatically, and Cachi2 downloads the fetched dependencies to your `deps/npm/` output directory as ``.tar` files.
```

Comment:
IMO it's not necessary to mention the `deps/npm/` directory. The Cachi2 functionality is integrated into the build pipeline, the user does not need to know any specifics of where Cachi2 downloads things

Maybe something like this would be enough: "Cachi2 fetches the dependencies and enables the build to install them with no network access"